lib/matplotlib/artist.py

```diff
@@ -1200,6 +1200,13 @@ def format_cursor_data(self, data):
 
     @property
     def mouseover(self):
+        """
+        If this property is set to *True*, the data from the artist will be 
```

Comment:
It looks like there is some trailing whitespace on this line, which our code checker doesn't like (https://travis-ci.com/matplotlib/matplotlib/jobs/254127846#L2418) - if you apply the suggestion in this comment it should fix things.
```suggestion
        If this property is set to *True*, the data from the artist will be
```

Reply:
"the data from the artist" feels a bit too unspecific. I'd rather go with something like
~~~
        Whether the artist should be queried for custom context information
        when the mouse cursor moves over it.
~~~

This also makes it more clear that the Artist has to provide the information (and it's not just a position or similar).

Reply:
I took your good suggestion into account and I reformulated liked this:
`If this property is set to *True*, the artist will be queried for custom context information when the mouse cursor moves over it.`